c2corg_ui/static/partials/imageuploader.html

```diff
@@ -0,0 +1,117 @@
+  <div ng-if="uplCtrl.files.length === 0"
+       ngf-drop
+       ngf-select
+       ng-model="uplCtrl.files"
+       class="drop-box" 
+       ngf-drag-over-class="'dragover'"
+       ngf-multiple="true"
+       ngf-keep="'distinct'" 
+       ngf-fix-orientation="true"
+       ngf-allow-dir="true"
+       accept="image/*" 
+       ngf-pattern="'image/*'" translate>
+    <span translate>Drop images here or click to upload</span>
+  </div>
+
+  <div ng-show="uplCtrl.files.length > 0" class="images-container" app-loading>
+    <div ng-repeat="file in uplCtrl.files track by $index" class="uploaded-image {{$index}}" ng-show="file">
```

Comment:
Is it possible to use an expression in ``class``? Doesn't it requires using ``ng-class``?